package.json

```diff
@@ -81,6 +81,9 @@
     "graphql-tag": "2.10.3",
     "i18next": "19.4.4",
     "i18next-locize-backend": "4.0.8",
+    "imagemin-mozjpeg": "^8.0.0",
```

Comment:
I think all deps should be devDeps, they're only used during build, right?

It helps devs to know what's actually being used at runtime.

Reply:
Yeah, that's true, my mistake.